source/extensions/filters/network/dubbo_proxy/router/route_matcher.h

```diff
@@ -0,0 +1,173 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "envoy/config/filter/network/dubbo_proxy/v2alpha1/dubbo_proxy.pb.h"
+#include "envoy/config/filter/network/dubbo_proxy/v2alpha1/route.pb.h"
+
+#include "common/common/logger.h"
+#include "common/http/header_utility.h"
+#include "common/protobuf/protobuf.h"
+
+#include "extensions/filters/network/dubbo_proxy/metadata.h"
+#include "extensions/filters/network/dubbo_proxy/router/router.h"
+
+#include "absl/types/optional.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace DubboProxy {
+namespace Router {
+
+class Utility {
```

Comment:
I prefer using an anonymous namespace

Reply:
I have moved utility class to utiliti.h file.